src/FormCheck.js

```diff
@@ -93,6 +97,7 @@ class FormCheck extends React.Component {
         {...props}
         type={type}
         ref={inputRef}
+        custom={custom}
```

Comment:
I think may make sense to put `custom` on the Context object below with controlId, that way when using the individual components you don't need to explicitly pass `custom` to the input and label

Reply:
Great feedback! I wasn't aware of the Context option. This helps to reduce code and makes the customize render option a lot easier to use.